Lesson1/Dima_Gnatyk/tests.cpp

```diff
@@ -0,0 +1,19 @@
+#include "main.cpp"
+#include <gtest/gtest.h>
+
+template <class T>
+std::vector<T> concat(const std::vector<T> &data_vector_1, const std::vector<T> &data_vector_2);
+
+namespace {
+class TestConcat : public ::testing::Test{
```

Comment:
Please move it to separated file. It is just a good practice to separate test suits with different files.